chord_diagram.py

```diff
@@ -21,7 +21,7 @@
 
 def chord_diagram(mat, names=None, order=None, width=0.1, pad=2., gap=0.03,
                   chordwidth=0.7, ax=None, colors=None, cmap=None, alpha=0.7,
-                  use_gradient=False, show=False, **kwargs):
+                  use_gradient=False, show=False, show_zeros=True, **kwargs):
```

Comment:
Thinking about it, I think it is definitely reasonable to make ``show_zeros=False`` the default behavior (I know of no cases were people would expect to see non-existent connections)

Reply:
Yeah, default should be zero. It's just I did not want to change API behavior in the PR.

Reply:
You have my blessing to do so, I'll also let you decide whether it should go into ``kwargs`` or not... I have no strong opinion on the matter